code/datums/traits/good.dm

```diff
@@ -108,3 +108,16 @@
 	mob_trait = TRAIT_VORACIOUS
 	gain_text = "<span class='notice'>You feel HONGRY.</span>"
 	lose_text = "<span class='danger'>You no longer feel HONGRY.</span>"
+/datum/trait/hlumi_eyes
+	name = "High Luminosity Eyes"
+	desc = "You decided to upgrade your weak fleshy eyes with these cool cybernetic ones, your vision is augumented!"
```

Comment:
Grammar here should be changed to match other traits and be more descriptive, like:
`Your eyes have been cybernetically enhanced, causing them to emit light in a small cone in front of you.`

Reply:
That doesn't carry the tone of "the user is implied to be unaware these ~hip transhuman eyes~ are actually a trashy cosmetic".